lib/urlwatch/reporters.py

```diff
@@ -610,10 +610,13 @@ def submitToTelegram(self, bot_token, chat_id, text):
 
 class SlackReporter(TextReporter):
     """Send a message to a Slack channel"""
-    MAX_LENGTH = 40000
 
     __kind__ = 'slack'
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.MAX_LENGTH = self.config.get('max_message_length', 40000)
```

Comment:
Sorry to nitpick again, but since it's not a constant anymore, prefer lowercase:

    self.max_length = ...

This also means changing all other occurences of `self.MAX_LENGTH` to `self.max_length`.

After that, it's ready to merge.